subprojects/diagnostics/src/test/groovy/org/gradle/api/tasks/diagnostics/PropertyReportTaskTest.groovy

```diff
@@ -27,76 +26,64 @@ class PropertyReportTaskTest extends Specification {
     @Rule
     final TestNameTestDirectoryProvider temporaryFolder = new TestNameTestDirectoryProvider(getClass())
 
-    private ProjectInternal project
     private PropertyReportRenderer renderer
     private PropertyReportTask task
 
     def setup() {
-        project = Mock()
         renderer = Mock()
         task = TestUtil.create(temporaryFolder).task(PropertyReportTask.class)
         task.setRenderer(renderer)
     }
 
     def "passes each project property to renderer"() {
-        given:
-        project.properties >> ['b': 'value2', 'a': 'value1']
```

Comment:
❓ aren't we losing coverage for project properties this way?

Reply:
I'd say it is not the intent of this test and that the fact that `project.getProperties()` includes _project properties_ is already covered. Moreover, `PropertyReportTaskIntegrationTest` also covers it.